Docs/schedule.md

```diff
@@ -1,15 +1,10 @@
-You can set a schedule for your Imgbot PRs through the `.imgbotconfig` file.
+The default behavior is to receive Imgbot pull requests as soon as images require optimization. To see less frequent pull requests, you can set an optional schedule for your Imgbot pull requests through the `.imgbotconfig` file. 
 
- - This configuration is optional and is only required if you want less frequent pull requests
- - Accepts the following syntax: daily|weekly|monthly
- - Limits the PRs from Imgbot to once a day, once a week, or once a month respectively
- - The default behavior is to receive Imgbot PRs as images require optimization
+Set the maximum pull request frequency submitted from Imgbot to your repo if your project receives many image updates and you don't want to see optimizations until they can all be batched up together. Most projects will benefit the most from the default setting of receiving image optimizations from Imgbot as soon as new images are added to the project.
```

Comment:
long sentence

Reply:
```suggestion
This will set the maximum pull request frequency submitted from Imgbot to your repo. This helps if your project receives many image updates and you don't want to see optimizations until they can all be batched up together. Most projects will benefit the most from the default setting for image optimizations when new images are added to the project.
```